Shared/Extensions/ViewExtensions/ViewExtensions.swift

```diff
@@ -46,12 +46,28 @@ extension View {
         }
     }
 
+    func poster(type: PosterType, width: CGFloat) -> some View {
```

Comment:
If it's that name, I feel like It's going to return the `poster` itself.
What about naming like `posterStyle`?

Reply:
I'll do `posterStyle` in future work so that the change won't remove your approval